src/Microsoft.Health.Dicom.Core/Features/Indexing/ReindexDatasetValidator.cs

```diff
@@ -35,7 +35,7 @@ public IReadOnlyCollection<QueryTag> Validate(DicomDataset dataset, long waterma
                     dataset.ValidateQueryTag(queryTag, _minimumValidator);
                     validTags.Add(queryTag);
                 }
-                catch (DicomElementValidationException)
+                catch (ElementValidationException)
                 {
                     // TODO: log failure
```

Comment:
I take it we'll resolve this in another PR?

Reply:
I remember we  need to remove `Dicom` prefix from all classes in this repository, but don't remember if I did this in another PR. :)